rmi/servercode/TransImpl/HashVault.java

```diff
@@ -0,0 +1,81 @@
+package TransImpl;
```

Comment:
One object, Hashtable of Hashtables.
This is what we shadow, as the "shadowVault.ser" file.